### PR TITLE
Tania development

### PR DESCRIPTION
**Pull request by tatounel**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -102,15 +102,26 @@
 
   </dependencies>
 
-  <build>
+  <!-- <build>
```
Comment (amnarules) [inline pom.xml]: Remove comment?

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <label for="aboutme"></label>
+      <br/>
+      <textarea rows="10" cols="100" name="aboutme" id="aboutme"></textarea>
+      <br/><br/>
+      <button type="submit">Submit</button>
+    </form>
+
+    <h2>Sent Messages</h2>
+    <div class="messages">
+              <%-- <% for (MessageStore i : messagesInConversation) {%>
```
Comment (amnarules) [inline src/main/webapp/WEB-INF/view/profile.jsp]: Remove comment?

File: pom.xml [hunk 3]
```diff
@@ -102,15 +102,26 @@
...
       </plugin>
     </plugins>
+  </build> -->
+  <build>
+      <plugins>
+            <plugin>
+              <groupId>com.google.cloud.tools</groupId>
+              <artifactId>appengine-maven-plugin</artifactId>
+              <version>1.3.1</version>
+              <configuration>
```
Comment (amnarules) [inline pom.xml:112]: Nice! 👍

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+          }
+
+          User user = userStore.getUser(username);
+
+          if(user == null){
+            response.sendRedirect("/login");
+            return;
+          }
+          String requestUrl = request.getRequestURI();
+          String userProfile = requestUrl.substring("/user/".length());
```
Comment (amnarules) [inline src/main/java/codeu/controller/ProfileServlet.java]: Is userProfile different from user.getName()?

**Review by amnarules — COMMENTED** [review]
Brilliant work Tania! 🎊Have you thought about adding tests for this?

**Review by amnarules — APPROVED** [review]

**Comment by zephyllr** [comment]
for future reference: try bulk committing as oppose to committing every few lines. that way it will be much easier to keep track of major changes. it is also good for reverting to a certain commit (if you ever needed to), but that will be very hard to do if your commits share the same commit comments. before you commit, you can do "git add -u" so that you can add all modified files after finishing up a feature or major change.

**Pull request merged**